package.json

```diff
@@ -23,8 +18,7 @@
     "react-scripts": "5.0.1",
     "react-select": "^5.7.0",
     "react-to-print": "^2.14.10",
-    "styled-components": "^5.3.6",
-    "web-vitals": "^2.1.4"
+    "styled-components": "^5.3.6"
```

Comment:
This package is also unused.

Reply:
styled components are used for data-table structuring.. right?

Reply:
could you tell me where it is used in the project? I need to take a look.

Reply:
D:\automotive\Automotive_Revamped\src\pages\EmployeeSheetDisplay\EmployeeSheetDisplay.js